src/gui/gtk.c

```diff
@@ -892,15 +892,6 @@ static gboolean mouse_moved(GtkWidget *w, GdkEventMotion *event, gpointer user_d
     gdk_event_get_axis ((GdkEvent *)event, GDK_AXIS_PRESSURE, &pressure);
   }
   dt_control_mouse_moved(event->x, event->y, pressure, event->state & 0xf);
-  gint x, y;
```

Comment:
Why removed that? Is that expected?

Reply:
This and the following half a dozen lines are essentially dead code (actually it runs, but it merely wastes cpu cycles). Specifically, it defines local variables x and y, then it fetches the pointer position into x and y, then, finally, it does return FALSE.

Reply:
Indeed, I don't see why this was needed! And it seems that it has been added long time ago.